conda/misc.py

```diff
@@ -50,7 +50,10 @@ def conda_installed_files(prefix, exclude_self_build=False):
 url_pat = re.compile(
     r"(?:(?P<url_p>.+)(?:[/\\]))?"
     r"(?P<fn>[^/\\#]+(?:\.tar\.bz2|\.conda))"
-    r"(:?#(?P<md5>[0-9a-f]{32}))?$"
+    r"(:?#("
```

Comment:
```suggestion
    r"(?:#("
```

This must be a typo. I haven't ever seen a `<url>:#<md5>` lockfile.